hypemaths/models/matrix.py

```diff
@@ -8,13 +8,16 @@ def __init__(
             self, matrix: t.Union[int, float, list] = None, dims: tuple = None, fill: t.Union[int, float] = None
     ) -> None:
         if not matrix:
-            if not dims or fill:
+            if not dims or fill is not None:
```

Comment:
```suggestion
            if not dims or fill:
```
Actually `not dims` auto checks if its None or not :) So we wont need it

Reply:
But it won't work with `fill=0` case

Reply:
Ohh, Alright then :)